test/python/opflow/test_aer_pauli_expectation.py

```diff
@@ -258,6 +255,13 @@ def test_pauli_expectation_non_hermite_op(self):
         exp = ~StateFn(1j * Z) @ One
         self.assertEqual(self.sampler.convert(self.expect.convert(exp)).eval(), 1j)
 
+    def test_expectation_with_coeff(self):
+        """Test AerPauliExpectation with coefficients."""
+        exp = 3 * ~StateFn(Z) @ (2 * One)
+        expected = exp.eval()
```

Comment:
Just a minor comment, if the expected answer is easily known would it not be better to hard code it here? The assert as it stands, i.e. equal to exp.eval() could be in error and equally wrong - so things would be self consistent in opflow but not what we really expect.

Reply:
Thanks. If both are broken at the same time, this test won't detect it. I will change to hard code here.